### ISSUE TITLE
Message printed twice: "Migrating existing metadata to the new RethinkDB 1.16 format..."

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#3467]
When migrating a 1.15 database to 1.16, the message "Migrating existing metadata to the new RethinkDB 1.16 format..." is printed twice:
```
Running rethinkdb foo (CLANG 3.2 (tags/RELEASE_32/final))...
Running on Linux 3.2.0-74-generic x86_64
Loading data from directory /home/ssd3/daniel/user-data/3463-data
Migrating existing metadata to the new RethinkDB 1.16 format...
Migrating existing metadata to the new RethinkDB 1.16 format...
Listening for intracluster connections on port 29038
Listening for client driver connections on port 28038
Listening for administrative HTTP connections on port 8103
Listening on addresses: 127.0.0.1, 127.0.1.1, 10.0.3.1, 192.168.122.1, 192.168.0.7, 192.168.1.7, ::1, fe80::225:90ff:fe0d:8e8a%2, fe80::225:90ff:fe0d:8e8b%3, fe80::5cec:cdff:febf:3601%4
Server ready, "x_space_vkr" b210256b-d8c5-4f8d-9544-e39d8f7b5cbb
```

**Comment by Tryneus** [comment]
Fix is up in review 2515.  It wasn't actually one for each file, but rather due to the nature of `persist.cc`, it read the file once to get the initial value, then again when updating the value.  The auth metadata migration was not logged.

Changed to only log the notice once for each file, but to also log the migration of the auth metadata file.  Users should now see:
```
Migrating existing cluster metadata to the new RethinkDB 1.16 format...
Migrating existing auth metadata to the new RethinkDB 1.16 format...
```

**Comment by Tryneus** [comment]
Fix approved and merged to `next` in commit 3b8ae19, and cherry-picked into `v1.16.x` in commit e2f5645.  Will be in release 1.16.0.
